src/main/java/com/superzanti/serversync/GUIJavaFX/Gui_JavaFX.java

```diff
@@ -30,6 +30,7 @@ public void start(Stage primaryStage) throws Exception {
         Scene scene = new Scene(root, 1024, 576);
         scene.getStylesheets().add(this.getClass().getResource("/css/application.css").toExternalForm());
         primaryStage.setScene(scene);
+        primaryStage.sizeToScene();
```

Comment:
Unrelated to logging?

Reply:
Yep, unrelated. This was done with version 3 in mind and an assumption that UI hasn't changed, as on Macs it was pretty darn tiny, like 300px wide or something. UI in v4 is much bigger now, thanks. But is it resizeable, though?..

Reply:
I believe it is user resizable, we can revisit the default size in a different pull request though if you would like.